### DISCUSSION TITLE
Feature request: Deno + SvelteKit + Tauri project

### DISCUSSION THREAD

**Discussion by keanesnow98** [denoland/deno#20511] [Ideas]
Hello, everyone!
I have a SvelteKit + Tauri project for building a robust and secure desktop application.
I have visited Deno recently and found out that it's also based on Rust.
I thought about migrating my current application using Bun but I think since it's using Zig, I would be smart choice to migrate using Deno, because it's Rust based!

But I am new to Deno, but there are less communities and examples than Node.js.
Can anyone help my migration stuff?
Thanks in advance!

**Comment by Cassielxd** [comment]
https://gitee.com/stringlxd/tauri-cc
